lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -585,6 +585,19 @@ def _determine_lims(self, xmin=None, xmax=None, *args, **kwargs):
             xmax += 0.05
         return (xmin, xmax)
 
+    def _validate_axis_limits(self, limit, convert):
+        """
+        If the axis limits being set are infinite, this function
+
+        raises an error.
+
+        """
+        if limit is not None:
+            if (isinstance(limit, float) and
+                    (not np.isreal(limit) or not np.isfinite(limit))):
+                raise ValueError("NaN or Inf cannot be the argument values")
```

Comment:
Users aren't going to know what "argument values" means; should say "Axis limits cannot be ..."